_posts/2016/2016-05-25-spring-io.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Spring I/O 2016"
+date: "2016-05-25 09:00:00"
+image: blog-header/springio.jpg
+categories: events
+authors: ["Xavi"]
+---
+Last 19th and 20th of May a conference called *Spring I/O* took place in Barcelona. It was the 5th edition of an event whose goal is to spread the word about the well-known framework [Spring](https://spring.io/). Luckily, four *epagees* including myself had the chance to attend the conference.
```

Comment:
Last 19th and 20th of May a conference called *Spring I/O* took place in Barcelona. > On 19th and 20th of May, the *Spring I/O* conference took place in Barcelona.

Luckily, four *epagees* including myself had the chance to attend the conference. > Luckily, four ePagees, including myself, had the chance to attend the conference.